### PR TITLE
added option to disable groups from the main FAQ view 

### PR DESCRIPTION
**Pull request by JopKnoppers**
- also create setting to turn this on and off

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
-Version Information
-===================
+Version history
+===============
+
+2.0.1 (2016-12-12):
```
Comment (jeabakker) [inline CHANGES.txt]: why is this file included in the PR

File: README.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ User Support
 
 [![Build Status](https://scrutinizer-ci.com/g/ColdTrick/user_support/badges/build.png?b=master)](https://scrutinizer-ci.com/g/ColdTrick/user_support/build-status/master)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/ColdTrick/user_support/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/ColdTrick/user_support/?branch=master)
+[![Latest Stable Version](https://poser.pugx.org/coldtrick/user_support/v/stable.svg)](https://packagist.org/packages/coldtrick/user_support)
```
Comment (jeabakker) [inline README.md]: why is this file included in the PR

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+		"composer/installers": "^1.0.8"
+	},
+	"require-dev": {
+		"coldtrick/sniffs": "dev-master",
+		"coldtrick/releases": "dev-master"
+	},
+	"conflict": {
+		"elgg/elgg": "<1.9"
+	}
+}
```
Comment (jeabakker) [inline composer.json]: why is this file included in the PR

File: manifest.xml [hunk 4]
```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <plugin_manifest xmlns="http://www.elgg.org/plugin_manifest/1.8">
 	<name>User Support</name>
+	<id>user_support</id>
 	<author>ColdTrick IT Solutions</author>
-	<version>2.0</version>
+	<version>2.0.1</version>
```
Comment (jeabakker) [inline manifest.xml]: why is this file included in the PR

File: pages/faq/list.php [hunk 5]
```diff
@@ -12,14 +12,19 @@
 	"type" => "object",
 	"subtype" => UserSupportFAQ::SUBTYPE,
 	"full_view" => false,
-	"metadata_name_value_pairs" => array(),
+	"metadata_name_value_pairs" => array(
+    ),
```
Comment (jeabakker) [inline pages/faq/list.php]: why this change

File: views/default/plugins/user_support/settings.php [hunk 6]
```diff
@@ -113,3 +113,8 @@
 )) . "</label><br />";
 
 echo elgg_view_module("inline", elgg_echo("user_support:settings:other:title"), $other);
+
+$show_groups = "<label>" . elgg_echo("user_support:show:group") . "&nbsp;";
+$show_groups .= elgg_view("input/select", array("name" => "params[show_group_faq]", "value" => $plugin->show_group_faq, "options_values" => $noyes_options)) . "</label><br />";
```
Comment (jeabakker) [inline views/default/plugins/user_support/settings.php]: for readability please change this to 
```php
$show_groups .= elgg_view("input/select", array(
          "name" => "params[show_group_faq]", 
          "value" => $plugin->show_group_faq, 
          "options_values" => $noyes_options,
)) . "</label><br />";
```

File: CHANGES.txt [hunk 7]
```diff
@@ -75,4 +75,4 @@ Version history
 
 1.0:
 
-- first release
\ No newline at end of file
+- first release
```
Comment (jeabakker) [inline CHANGES.txt]: this file shouldn't be in the PR

**Comment by jdalsem** [comment]
Closing this, as the PR became stale
